src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -1076,7 +1076,8 @@ private bool CheckPropertyValueKind(SyntaxNode node, BoundExpression expr, BindV
                 if (setMethod is null)
                 {
                     var containing = this.ContainingMemberOrLambda;
-                    if (!AccessingAutoPropertyFromConstructor(receiver, propertySymbol, containing))
+                    if (!AccessingAutoPropertyFromConstructor(receiver, propertySymbol, containing)
+                        && !isAllowedReadonlyOnlySet(receiver))
```

Comment:
>isAllowedReadonlyOnlySet [](http://example.com/codeflow?start=28&length=24)

Is it intentional to use "only" twice? Not sure what "OnlySet" is supposed to convey.

Reply:
I suggest inlining the condithin and avoiding all the naming "problem"